templates/$package.json

```diff
@@ -18,5 +18,8 @@ rename:
   "license": "<%= ask('license', {default: 'MIT'}) %>",
   "scripts": {
     "test": "mocha"
-  }
+  },
+  "keywords": [
+    "generategenerator"
+  ]
```

Comment:
@dawsonbotsford just thought about this more. Instead of doing this here, we should add this keyword in `generate-generator`. I'll move this over - this was a good idea though